cmd/main.go

```diff
@@ -404,6 +404,13 @@ func setupManifestReconciler(mgr ctrl.Manager, flagVar *flags.FlagVar, options c
 			Busy:    flagVar.ManifestRequeueBusyInterval,
 			Error:   flagVar.ManifestRequeueErrInterval,
 			Warning: flagVar.ManifestRequeueWarningInterval,
+			Jitter: &queue.RequeueJitter{
+				Enabled:           true,
```

Comment:
I don't think we need an "Enabled" attribute. I don't even think we need such property (method). If the jitter value is zero we can just "apply it" - zero jitter means no jitter. So the value of zero for JitterPercentage naturally disables it, and anything greater than zero enables it.